arangod/Aql/JoinExecutor.cpp

```diff
@@ -61,14 +61,14 @@ void JoinExecutorInfos::determineProjectionsForRegisters() {
 JoinExecutor::~JoinExecutor() {
   // handle decrease of memory usage of the projections builder
   if (!_projectionsBuilder.isEmpty()) {
-    resourceMonitor().decreaseMemoryUsage(_projectionsBuilder.size());
+    // resourceMonitor().decreaseMemoryUsage(_projectionsBuilder.size());
```

Comment:
I suggest cleaning this (and following occurrences) up to make it easier to follow where memory management happens.